src/content/docs/how-to-contribute-to-the-codebase.mdx

```diff
@@ -238,7 +238,7 @@ A quick reference to the commands that you will need when working.
 | `pnpm run test-client -u`                                            | Run the client test suite, updating the Jest snapshots that are out of sync.        |
 | `pnpm run test:curriculum`                                           | Run the curriculum test suite.                                                      |
 | `FCC_CHALLENGE_ID=646cf6cbca98e258da65c979 pnpm run test:curriculum` | Test a specific Challenge.                                                          |
-| `FCC_BLOCK='Basic HTML and HTML5' pnpm run test:curriculum`          | Test a specific Block.                                                              |
+| `FCC_BLOCK='basic-html-and-html5' pnpm run test:curriculum`          | Test a specific Block.                                                              |
```

Comment:
now it matches
https://github.com/freeCodeCamp/contribute/blob/d46f33b4ca335af2ab0f2af940815a676e01696f/src/content/docs/how-to-work-on-coding-challenges.mdx#L680-L687

The test should work the same, but it's less likely to test something else if you use the exact dashedName